### PR TITLE
2

### PR DESCRIPTION
**Pull request by marcbloc**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
+﻿<!DOCTYPE html>
+<html lang="pl">
+<head>
+	<neta charset="UTF-8">
```
Comment (matwjablonski) [inline index.html]: `<neta>` not exist in html doc

File: index.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" />
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.min.css" />
+	<title>The game -Javascript</title>
+	</head>
```
Comment (matwjablonski) [inline index.html]: one tab to remove

File: index.html [hunk 3]
```diff
@@ -0,0 +1,79 @@
+﻿<!DOCTYPE html>
+<html lang="pl">
+<head>
+	<neta charset="UTF-8">
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" />
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.min.css" />
```
Comment (matwjablonski) [inline index.html]: on end of tag link you don't need to add `/`

File: js/scripts.js [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+	playerPickElem.innerHTML = playerPick; 
+	computerPickElem.innerHTML = computerPick; 
+}
+
+function checkRoundWinner(playerPick, computerPick) { 
+	playerResultElem.innerHTML = computerResultElem.innerHTML = ''; 
+	
+	var winnerIs = 'player'; 
+	
+		if (playerPick == computerPick) {
```
Comment (matwjablonski) [inline js/scripts.js]: better will be compare by `===` instead of `==`

File: js/scripts.js [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+
+function checkRoundWinner(playerPick, computerPick) { 
+	playerResultElem.innerHTML = computerResultElem.innerHTML = ''; 
+	
+	var winnerIs = 'player'; 
+	
+		if (playerPick == computerPick) { 
+			winnerIs = 'noone'; // remis 
+		} else if ( 
+			(computerPick == 'rock' && playerPick == 'scissors') ||
```
Comment (matwjablonski) [inline js/scripts.js]: `===` instead of `==`

File: js/scripts.js [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+function checkRoundWinner(playerPick, computerPick) { 
+	playerResultElem.innerHTML = computerResultElem.innerHTML = ''; 
+	
+	var winnerIs = 'player'; 
+	
+		if (playerPick == computerPick) { 
+			winnerIs = 'noone'; // remis 
+		} else if ( 
+			(computerPick == 'rock' && playerPick == 'scissors') || 
+			(computerPick == 'scissors' && playerPick == 'paper') ||
```
Comment (matwjablonski) [inline js/scripts.js]: `===` instead of `==`

File: js/scripts.js [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+	playerResultElem.innerHTML = computerResultElem.innerHTML = ''; 
+	
+	var winnerIs = 'player'; 
+	
+		if (playerPick == computerPick) { 
+			winnerIs = 'noone'; // remis 
+		} else if ( 
+			(computerPick == 'rock' && playerPick == 'scissors') || 
+			(computerPick == 'scissors' && playerPick == 'paper') || 
+			(computerPick == 'paper' && playerPick == 'rock')) {
```
Comment (matwjablonski) [inline js/scripts.js]: `===` instead of `==`

File: js/scripts.js [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+			winnerIs = 'noone'; // remis 
+		} else if ( 
+			(computerPick == 'rock' && playerPick == 'scissors') || 
+			(computerPick == 'scissors' && playerPick == 'paper') || 
+			(computerPick == 'paper' && playerPick == 'rock')) { 
+				
+			winnerIs = 'computer'; 
+		} 
+
+		if (winnerIs == 'player') {
```
Comment (matwjablonski) [inline js/scripts.js]: `===` instead of `==`

File: js/scripts.js [hunk 9]
```diff
@@ -0,0 +1,130 @@
...
+			(computerPick == 'scissors' && playerPick == 'paper') || 
+			(computerPick == 'paper' && playerPick == 'rock')) { 
+				
+			winnerIs = 'computer'; 
+		} 
+
+		if (winnerIs == 'player') { 
+			playerResultElem.innerHTML = "Wygrana!"; 
+			player.score++; 
+		} else if (winnerIs == 'computer') {
```
Comment (matwjablonski) [inline js/scripts.js]: `===` instead of `==`

File: js/scripts.js [hunk 10]
```diff
@@ -0,0 +1,130 @@
...
+	checkRoundWinner(playerPick, computerPick); 
+}
+
+function setGamePoints() { 
+	playerPointsElem.innerHTML = player.score; 
+	computerPointsElem.innerHTML = computer.score; 
+}
+
+function gameOver() {
+	if (player.score == 10) {
```
Comment (matwjablonski) [inline js/scripts.js]: `===` instead of `==`

File: js/scripts.js [hunk 11]
```diff
@@ -0,0 +1,130 @@
...
+		alert("Wygrał " + player.name)
+	} else if (computer.score == 10) {
```
Comment (matwjablonski) [inline js/scripts.js]: `===` instead of `==`

**Review by matwjablonski — CHANGES_REQUESTED** [review]

**Comment by marcbloc** [comment]
corrected

File: js/scripts.js [hunk 12]
```diff
@@ -0,0 +1,129 @@
...
+
+function checkRoundWinner(playerPick, computerPick) { 
+	playerResultElem.innerHTML = computerResultElem.innerHTML = ''; 
+	
+	var winnerIs = 'player'; 
+	
+		if (playerPick === computerPick) { 
+			winnerIs = 'noone'; // remis 
+		} else if ( 
+			(computerPick === 'rock' && playerPick == 'scissors') ||
```
Comment (matwjablonski) [inline js/scripts.js]: second statement also should be changed :)

**Review by matwjablonski — CHANGES_REQUESTED** [review]

**Review by matwjablonski — APPROVED** [review]

**Pull request merged**